listenbrainz/webserver/static/js/src/types.d.ts

```diff
@@ -32,20 +32,25 @@ interface AdditionalInfo {
   work_mbids?: Array<string> | null;
 }
 
-declare type Listen = {
+declare type BaseListenFormat = {
   listened_at: number;
-  listened_at_iso?: string | null;
-  playing_now?: boolean | null;
   user_name?: string | null;
   track_metadata: {
     artist_name: string;
     release_name?: string | null;
     track_name: string;
     additional_info?: AdditionalInfo;
   };
+};
+
+declare type Listen = BaseListenFormat & {
+  listened_at_iso?: string | null;
+  playing_now?: boolean | null;
   score?: number;
```

Comment:
I wonder if this is relevant anywhere.
Doesn't need to be addressed in this PR, but maybe have a closer look at it when we tackle listened_at/BP changes.